README.md

```diff
@@ -0,0 +1,424 @@
+## Luxonis Dataset Format
```

Comment:
Do you mind just adding Alpha Version notice at the top. Something saying like that the API might change.